Automattic/wp-calypso
# Jetpack Logo Generator: use local storage history

Keep a history of generated logos/images on localStorage

Related to #85557 

## Proposed Changes

* use localStorage to keep last 10 generated images
* trigger load history on modal open
* provide a default logo/description for logo presenter when history is empty


## Testing Instructions
Open the modal on one site, generate at least 1 logo.
Close modal and switch sites, open modal to see the logo history is reset
Close and switch back to first site, logo history should be there
